app/models/comment.rb

```diff
@@ -0,0 +1,14 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post
+
+  attribute :text, :text
+
+  after_create :increment_post_comments_counter
```

Comment:
- [ ] The `after_save` callback might indeed be a better choice in this scenario, as it's more flexible and versatile compared to `after_create`. 

By using `after_save`, you're allowing the same functionality to run after both `create` and `update` actions. This can be useful if you ever need to modify this behavior in the future. Additionally, marking the method as private ensures that it's only accessible from within the class and can't be called externally, providing better encapsulation and maintaining the principle of least privilege.

```suggestion

  after_save :increment_post_comments_counter
```